pkgs/development/python-modules/python-chess/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "python-chess";
```

The package name should match the [name](https://github.com/niklasf/python-chess/blob/master/setup.py#L78). It helps if people are checking [PyPI](https://pypi.org/project/chess/) for details.

```suggestion
  pname = "chess";
```

---

I had it originally as "chess", but then I got confused because in the webpage and the readme it is named "python-chess". I moved it to just "chess" then.